syntax = "proto3";
package cavs;

enum DataType {
    DT_FLOAT    = 0;
    DT_DOUBLE   = 1;
    DT_INT32    = 2;
}
